earth2studio/models/px/aifs.py

```diff
@@ -0,0 +1,857 @@
+    "t150",
+    "t200",
+    "t250",
+    "t300",
+    "t400",
+    "t500",
+    "t600",
+    "t700",
+    "t850",
+    "t925",
+    "t1000",
+    "u50",
+    "u100",
+    "u150",
+    "u200",
+    "u250",
+    "u300",
+    "u400",
+    "u500",
+    "u600",
+    "u700",
+    "u850",
+    "u925",
+    "u1000",
+    "v50",
+    "v100",
+    "v150",
+    "v200",
+    "v250",
+    "v300",
+    "v400",
+    "v500",
+    "v600",
+    "v700",
+    "v850",
+    "v925",
+    "v1000",
+    "w50",
+    "w100",
+    "w150",
+    "w200",
+    "w250",
+    "w300",
+    "w400",
+    "w500",
+    "w600",
+    "w700",
+    "w850",
+    "w925",
+    "w1000",
+    "z50",
+    "z100",
+    "z150",
+    "z200",
+    "z250",
+    "z300",
+    "z400",
+    "z500",
+    "z600",
+    "z700",
+    "z850",
+    "z925",
+    "z1000",
+    "u10m",
+    "v10m",
+    "d2m",
+    "t2m",
+    "lsm",
+    "msl",
+    "sdor",
+    "skt",
+    "slor",
+    "sp",
+    "tcwv",
+    "zsl",
+    "stl1",
+    "stl2",
+    "swvl1",
+    "swvl2",
+]
+
+##########################################3
+# Remove after debugging
+##########################################3
+# import matplotlib.pyplot as plt
+# import cartopy.crs as ccrs
+# import cartopy.feature as cfeature
+# import matplotlib.tri as tri
+#
+# def plot_field(latitudes, longitudes, values, title):
+#    def fix(lons):
+#        # Shift the longitudes from 0-360 to -180-180
+#        return np.where(lons > 180, lons - 360, lons)
+#    fig, ax = plt.subplots(figsize=(11, 6), subplot_kw={"projection": ccrs.PlateCarree()})
+#    ax.coastlines()
+#    ax.add_feature(cfeature.BORDERS, linestyle=":")
+#    triangulation = tri.Triangulation(fix(longitudes), latitudes)
+#    contour=ax.tricontourf(triangulation, values, levels=20, transform=ccrs.PlateCarree(), cmap="RdBu")
+#    cbar = fig.colorbar(contour, ax=ax, orientation="vertical", shrink=0.7, label="100u")
+#    plt.title(title)
+#    plt.show()
+
+
+class AIFS(torch.nn.Module, AutoModelMixin, PrognosticMixin):
+    """AIFS global prognostic model.
+    Consists of a single model with a time-step size of 6 hours.
+
+    Parameters
+    ----------
+    variables : np.array, optional
+        Variables associated with model, by default 73 variable model.
+    """
+
+    def __init__(
+        self,
+        model: torch.nn.Module,
+        latitudes: torch.Tensor,
+        longitudes: torch.Tensor,
+        interpolation_matrix: torch.Tensor,
+        inverse_interpolation_matrix: torch.Tensor,
+        variables: np.array = np.array(VARIABLES),
+    ) -> None:
+        super().__init__()
+        self.model = model
+        self.variables = variables
+        self.register_buffer("latitudes", latitudes)
+        self.register_buffer("longitudes", longitudes)
+        self.register_buffer("interpolation_matrix", interpolation_matrix)
+        self.register_buffer(
+            "inverse_interpolation_matrix", inverse_interpolation_matrix
+        )
+
+        # Mask indices
```

Comment:
Black formatter really does a number on these,
How are these determined btw? Seems its like 10 or 11 numbers are missing?

Reply:
I got these as a print out directly from the anemio package. Seems like they kinda randomly added variables at the end and this was the result.